### ISSUE TITLE
GLOBAL_PING not working

### ISSUE THREAD

**Issue by Prifabu** [SpigotMC/BungeeCord#2798]
Option GLOBAL_PING does not show every player in the tab_list, only the players on the current server

**Comment by Janmm14** [comment]
This is known since the release of minecraft 1.8 and likely won't change anytime soon due to how stuff is working technically.

**Comment by Nubebuster** [comment]
damn I need this tho

**Comment by Janmm14** [comment]
I'd suggest using or asking for or creating a spigot plugin which simulates this. Needs to be on all your spigot servers then and some way to communicate though.

**Comment by Nubebuster** [comment]
Yeah I'm considering that. Will post a link to the spigot resource and github if I make it

**Comment by MrIvanPlays** [comment]
Actually bungee only impements server unique tablist and not any other variants of it. You can use 3rd party plugins to make so every server's players are displayed.

**Comment by Nubebuster** [comment]
@MrIvanPlays The problem I ran into is that sending packets from bungeecord is unreliable when you have support for multiple minecraft protocol versions. 
I tried making my own plugin but messing with PLAYER_INFO can cause many obscure bugs. I ended up using the spigot plugin TabList on each backend and sending player info from the bungeecord using Redis.

It's not what I had hoped for but the results are pretty good!

**Comment by MrIvanPlays** [comment]
If you want multi version support just dont make a server please for the love of god.

**Comment by Nubebuster** [comment]
> If you want multi version support just dont make a server please for the love of god.

You're making little sense... Are you implying I should either go for one version support, or not make a server at all?

**Comment by MrIvanPlays** [comment]
> > If you want multi version support just dont make a server please for the love of god.
> 
> You're making little sense... Are you implying I should either go for one version support, or not make a server at all?

Multi version support is broken atm. It is fine for 1.8 - 1.12, but the breaking 1.13 changes make it harder to support both as spigot calls them "legacy" versions and the new 1.13+ versions. 

So if you make a multi version support server you either go 1.8 - 1.12 or 1.13 - latest. Or else I won't like you.
